.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         run: sudo apt-get install -y libxmlsec1-dev
 
       - name: Lints
-        run: ./pants lint "tests/mitol/${{ matrix.app-name }}:"
+        run: ./pants lint "src/mitol/${{ matrix.app-name }}:" "tests/mitol/${{ matrix.app-name }}:"
```

Comment:
An alternate here could be just running the lint on all the files overall e..g `./pants lint ::`. I just kept it scoped to `src` and `tests` directories.

Reply:
I think this is what we want because we want it to just run on that particular app.